### ISSUE TITLE
Float and Integer not working as plugins

### ISSUE THREAD

**Issue by JRSV** [plugdata-team/plugdata#354]
Hello, 

the float and integer objects are not working properly in plug data used as a plug-in (LV2 confirmed)
the object does not pass a new value, if the object is initialized with a stored value as an argument [float 5] then that number is output correctly, but if I want to store a new number and then output it with a bang, the object only outputs 0. The objects also pass value if you send a list and its truncated. Bellow a giff of Plug Data in Reaper showing the bug. 

![plugdata-bug](https://user-images.githubusercontent.com/18346930/212371166-04bbbe89-476f-4f42-a441-a5cfd20425ae.gif)

**Comment by timothyschoen** [comment]
Hi, it looks like you're still using v0.6.2, could you try [v0.6.3](https://github.com/plugdata-team/plugdata/releases/tag/v0.6.3)? 

I'm not able to reproduce it with v0.6.3:

![ezgif-1-2c6a341dbd](https://user-images.githubusercontent.com/44585538/212374569-475d7d83-bbdf-4241-9468-2ae32fca89bf.gif)

**Comment by JRSV** [comment]
hmmm.. I downloaded the latest installer, I don't know why it does not use the correct version... let me try again and I will get back to you

**Comment by timothyschoen** [comment]
That's weird, you're on Windows right? Maybe it didn't overwrite the previous version...

**Comment by JRSV** [comment]
no, linux and I just installed it through OBS packaging but the plugin did not update ... let me try grabbing the installer

**Comment by JRSV** [comment]
Yes!! thank you... it seems I had to update it individually. 0.63 works as expected, sorry for the noise.

**Comment by timothyschoen** [comment]
No problem, glad this is resolved!
